Java/FrequencyList.java

```diff
@@ -0,0 +1,40 @@
+package lfu;
```

Comment:
lfu is a bad package name. But your choice though. Given your codebase is likely to be small you can go with it.